src/schedulers.jl

```diff
@@ -71,10 +72,10 @@ struct DynamicScheduler{C <: ChunkingMode} <: Scheduler
     threadpool::Symbol
     nchunks::Int
     chunksize::Int
-    split::Symbol
+    split::Split
```

Comment:
Since this is an abstract type, should it be a parameter? Perhaps it doesn't matter much in practice though

Reply:
Good catch, didn't think of this when switching from `Symbol` to `Split`. Will change it.